lib/provider/data_types.go

```diff
@@ -105,3 +105,14 @@ type Snapshot struct {
 	// tags for the snapshot
 	SnapshotTags SnapshotTags `json:"tags"`
 }
+
+// AuthorizationRequest capture details of autorization to be made
+type AuthorizationRequest struct {
+	// Volume to update the authorization
+	Volume Volume
+	// List of subnets to authorize. It might be SubnetIDs or CIDR based on the providers implementaions
+	// For example, IBM Softlyaer provider  expects SubnetIDs to be passed
+	Subnets []string
+	// List of HostIPs to authorize
+	HostIps []string
```

Comment:
please change `HostIps` to `HostIPs` similar to the comment

Reply:
also can we have json tag as well for all struct filed similar to other struct

Reply:
done